agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -244,6 +248,37 @@ public AzureMonitorExporterBuilder credential(TokenCredential credential) {
    *     environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
    */
   public AzureMonitorTraceExporter buildTraceExporter() {
+    initExporterBuilder();
+    SpanDataMapper mapper =
+        new SpanDataMapper(
+            true,
+            builder -> {
+              builder.setInstrumentationKey(instrumentationKey);
+              builder.addTag(
+                  ContextTagKeys.AI_INTERNAL_SDK_VERSION.toString(),
+                  VersionGenerator.getSdkVersion());
+            },
+            (event, instrumentationName) -> false,
+            () -> null);
+
+    return new AzureMonitorTraceExporter(mapper, telemetryItemExporter);
+  }
+
+  /**
+   * Creates an {@link AzureMonitorMetricExporter} based on the options set in the builder. This
+   * exporter is an implementation of OpenTelemetry {@link MetricExporter}.
+   *
+   * @return An instance of {@link AzureMonitorMetricExporter}.
+   * @throws NullPointerException if the connection string is not set on this builder or if the
+   *     environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
+   */
+  public AzureMonitorMetricExporter buildMetricExporter() {
+    initExporterBuilder();
+    return new AzureMonitorMetricExporter(
+        new MetricDataMapper(instrumentationKey, t -> {}), telemetryItemExporter);
+  }
+
+  private void initExporterBuilder() {
```

Comment:
last comment, I think this might be clearer to return `TelemetryItemExporter` and not store it into "temporary" field

I'm also ok waiting and seeing how we feel about it over time, it's not public API so we can always change it

```suggestion
  private TelemetryItemExporter createTelemetryItemExporter() {
```

Reply:
Updated.